new_project/examples/btc-eth/README.md

```diff
@@ -0,0 +1,25 @@
+# Hello-Swap
```

Comment:
The full README should go to the root of the `new_project` folder.
Here, should be a one liner saying that in this example the maker buy X ETH for Y BTC from the taker.